index.html

```diff
@@ -12,206 +12,257 @@
+    <header class="main-header">
+      <nav class="nav">
+        <a href="#" class="logo">My Logo</a>
+        <img src="./media/Union.png" alt="Union" width="30px">
+        <ul>
           <li>
-            <a href="#">
-              <img src="media/medium.svg" alt="medium" />
-            </a>
+            <a href="#">Portfolio</a>
           </li>
           <li>
-            <a href="#">
-              <img src="media/github.svg" alt="github" />
-            </a>
+            <a href="#">About</a>
           </li>
           <li>
-            <a href="#">
-              <img src="media/angelist.svg" alt="angellist" />
-            </a>
+            <a href="#">Contact</a>
           </li>
         </ul>
+      </nav>
+
+    </header>
+    <!--Welcome section-->
+
+
+    <main class="main-section">
+
+      <div class="main-title">
+        <h1>I'm Gilbert, Glad to See You!</h1>
+
+        <p>
+          I'm a software developer! I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+
+        <div class="lets-connect"> LET'S CONNECT</div>
+        <!--End of Welcome section-->
+
+        <!-- Social Media  Welcome section-->
+        <div class="icons-container">
+          <ul class="social-media-icons">
+            <li>
+              <a href="#">
+                <img src="media/twitter.svg" alt="twitter" />
+
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="media/Linkedin-icon.svg" alt="linkedin" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="media/medium.svg" alt="medium" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="media/github.svg" alt="github" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="media/angelist.svg" alt="angellist" />
+              </a>
+            </li>
+          </ul>
+        </div>
       </div>
 
       <!--End of Social media Welcome section-->
-    </section>
-  </main>
+    </main>
+  </section>
 
   <!--These are my projects-->
 
-  <section class="Work-projects">
+  <section class="cards">
 
-    <div class="card">
-      <div class="projects-done">
-        <img src="./media/Snapshoot Portfolio.png" alt="Website" width="98%">
-        <h2> Tonic</h2>
+    <div class="cards-section">
 
-        <ul>
-          <li>CANOPY</li>
-          <li>Back End Dev</li>
-          <li>2015</li>
-        </ul>
+      <div class="two">
+        <img src="./media/Snapshoot Portfolio-d1.png" alt="Website" width="100%">
       </div>
 
-      <div class="container">
-        <p> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-      </div>
```

Comment:
- [ ] Please make sure you follow the figma template. kindly adjust your texts and button. 
## your design👇
![tonicst](https://user-images.githubusercontent.com/66992901/194417984-b1bcbd77-0e4c-4b74-9107-1f75c2b1180b.PNG)


## figma design👇
![cardfigma](https://user-images.githubusercontent.com/66992901/194417770-43cf8475-8c26-4e3f-a5ae-44dbbe09d798.PNG)